### ISSUE TITLE
Can't see or copy error from tooltip Karabiner shows when it stops working

### ISSUE THREAD

**Issue by dandv** [pqrs-org/Karabiner-Elements#3526]
Here's the screenshot. The error is truncated.

<img width="421" alt="image" src="https://github.com/pqrs-org/Karabiner-Elements/assets/33569/c97f0a06-43c2-4c10-b31f-70be045240cb">

That's all I see, and I can't copy/paste the message. There are many bug reports about this, but I can't tell which one is about the same error.:
 * #3513
 * #2640
 * #2641 (?)
 * #3386 (?)

Please make this tooltip resize automatically, and make its text selectable.

**Comment by adriel** [comment]
Did you find out the cause/fix to this?

I've just started getting this same error. Happened to me twice so far, both right after waking up from black screen/screen saver.

**Comment by GaurangTandon** [comment]
@adriel Did you find any fix for this issue? I have the same error and it appears each time on waking up from sleep/booting up.

**Comment by adriel** [comment]
Yeah I never could pinpoint the issue. The best I done was isolate it to the "Apple internal keyboard / trackpad" in the "Devices" menu.

At least for me, I disabled it and the issue went away. But it also means I can no longer use the hotkey changes I had setup on my Mac keyboard. But can't have everything right :P

<img width="330" alt="image" src="https://github.com/pqrs-org/Karabiner-Elements/assets/154217/028b5b26-8d57-4cc2-8fbc-f1349458f804">

Good luck.
